DESCRIPTION

```diff
@@ -29,6 +29,7 @@ Suggests:
     data.table,
     DiagrammeR,
     DiagrammeRsvg,
+    glue,
```

Comment:
I added this dependency to the `Suggests` array since that seems to be the pattern for development/testing dependencies.